scripts/build_package.py

```diff
@@ -639,9 +639,35 @@ def build_package(repository_root, build_package_config, env=None):
     time_now = time.strftime("%Y-%m-%dT%H:%M")
     version_json_data['package_build_date'] = time_now
 
+    version_json_data['available_commands'] = []
```

Comment:
The version file is the one which contains the _most_ build-time created information, such at git describe and the like.

I felt it is appropriate to put the information here as the list of available commands is a property of a particular build taken, ie. users could arbitrarily add or remove commands before building their package.